### PR TITLE
Updated versions for Storage STG75 Beta Release

### PR DESCRIPTION
**Pull request by gapra-msft**

### CODE REVIEW

File: eng/jacoco-test-coverage/pom.xml [hunk 1]
```diff
@@ -259,7 +259,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-internal-avro</artifactId>
-      <version>12.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-internal-avro;current} -->
+      <version>12.0.2-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-internal-avro;current} -->
```
Comment (samvaity) [inline eng/jacoco-test-coverage/pom.xml:262]: The beta series shouls be `12.1.0 ...`

Comment (gapra-msft) [reply eng/jacoco-test-coverage/pom.xml:262]: I think this is fine. We added no features to internal avro - Just updated the dependency on azure-storage-common

**Review by samvaity — APPROVED** [review]

File: sdk/storage/azure-storage-internal-avro/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
 # Release History
 
-## 12.1.0-beta.1 (Unreleased)
+## 12.0.2-beta.1 (2020-12-07)
+- Update `azure-storage-common` to version `12.10.0-beta.1`
 
+## 12.1.0-beta.1 (2020-12-07)
```
Comment (rickle-msft) [inline sdk/storage/azure-storage-internal-avro/CHANGELOG.md:6]: Seems like a duplicate?

**Review by rickle-msft — APPROVED** [review]

File: sdk/storage/azure-storage-internal-avro/CHANGELOG.md (hunk 2 above)
Comment (gapra-msft) [reply sdk/storage/azure-storage-internal-avro/CHANGELOG.md]: Good catch

**Pull request merged**